source/documentation/runbooks/004-analytical-platform-compute-maintenance.html.md.erb

```diff
@@ -0,0 +1,117 @@
+---
+owner_slack: "#analytical-platform-notifications"
+title: Analytical Platform Compute Maintenance - Draft
+last_reviewed_on: 2024-10-28
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+
+On the first day of the month the workflow [schedule-issue-compute-infrastructure.yml] (https://github.com/ministryofjustice/analytical-platform/blob/bba07f4908b443a2c15bf94738c8b84c5211e9f9/.github/workflows/schedule-issue-compute-infrastructure.yml) will automatically raise a ticket for [Maintenance - Analytical Platform Compute ] (https://github.com/ministryofjustice/analytical-platform/issues/5661 ).
```

Comment:
The hyperlinks will break I think, should be of the form `[]()` without spaces. Additionally I think you're missing the word "example" so the sentence reads "...will automatically raise a ticket for _example_...".

Reply:
amended